blueprints/ember-frost-sort/index.js

```diff
@@ -1,22 +1,33 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  description: '',
-  normalizeEntityName: function () {},
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-computed-decorators', target: '~0.3.0'},
+      {name: 'ember-frost-core', target: '^1.14.3'},
+      {name: 'ember-elsewhere', target: '~0.4.1'},
+      {name: 'ember-hook', target: '^1.4.1'},
+      {name: 'ember-prop-types', target: '^3.10.2'}
```

Comment:
They will, but this addon is also using those so I'm expecting that we need those here.